libs/designer-ui/src/lib/floatingactionmenu/floatingactionmenuoutputs/index.tsx

```diff
@@ -61,6 +63,15 @@ export const FloatingActionMenuOutputs = (props: FloatingActionMenuOutputsProps)
     }
   };
 
+  const onDynamicallyAddedParameterDescriptionChange = (schemaKey: string, newValue: string | undefined): void => {
+    const { onChange } = props;
+    if (onChange) {
+      const viewModel = clone(props.editorViewModel);
+      viewModel.schema.properties[schemaKey].description = newValue || '';
```

Comment:
```suggestion
      viewModel.schema.properties[schemaKey].description = newValue ?? '';
```
because of Javacripts truthy/falsy nature, it's better to use nullish coalescing for this as well

Reply:
I agree, and was following the pattern from the `onDynamicallyAddedParameterTitleChange` someone else wrote and didn't notice the antiquated operator, it now uses `??` as does the other spot